bash_completion.sh

```diff
@@ -216,7 +216,7 @@ _buildtest ()
       ;;
 
     buildspec|bc)
-      local cmds="-h --help edit edit-file find show summary validate"
+      local cmds="-h --help edit-test edit-file find show summary validate"
```

Comment:
This is correct place to make change! Good job. 

You can test bash completion by doing `source bash_completion.sh` and then `TAB` away :-)